modules/installation-special-config-crony.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-config-crony_{context}"]
+= Configuring chrony time service
+
+You can set the time server and related settings used by the chrony time service (chronyd)
+by modifying the contents of the `chrony.conf` file and passing those contents
+to your nodes as a MachineConfig.
+
+.Procedure
+
+. Create the contents of the `chrony.conf` file and encode it as base64. For example:
++
+----
+$ cat << EOF | base64
+    server clock.redhat.com iburst
```

Comment:
When I copy and paste this block into my terminal, it doesn't recognize the end EOF because of the spacing.  I noticed in the kmod-in-containers section this indentation is not used and makes it easier to copy and past.

Reply:
Okay. Yes, some of the spacing was off. I fixed it and was able to successfully include it in a manifest that I installed.